.gitattributes

```diff
@@ -1,7 +1,6 @@
-# Do not merge `pom.xml` from older version, as it will typically conflict
 #
-# note: also need:
-# git config --global merge.ours.driver true
-# (and if ever need to disable change `true` to `false`
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# These are explicitly windows files and should use crlf
+*.bat           text eol=crlf
 
```

Comment:
Please do not overwrite existing settings -- `pom.xml` conflicts are a big nuisance across branches.